Example/CustomActionControllers/Twitter.swift

```diff
@@ -94,10 +94,13 @@ open class TwitterActionController: ActionController<TwitterCell, ActionData, Tw
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
         settings.animation.present.duration = 0.6
         settings.animation.dismiss.duration = 0.6
+        settings.cancelView.hasCancelView = true
```

Comment:
Maybe we can get rid of this property and only check if `cancelView` was set. I think that will simplify some stuff